### ISSUE TITLE
typegen response format is unexpected

### ISSUE THREAD

**Issue by infomofo** [anttiviljami/openapi-client-axios#25]
I have an openapi json file with responses for operations defined like this:

```
 "responses": {
                    "404": {
                        "description": "Not found"
                    },
                    "422": {
                        "description": "Validation Error",
                        "content": {
                            "application/json": {
                                "schema": {
                                    "$ref": "#/components/schemas/HTTPValidationError"
                                }
                            }
                        }
...
```

When I run `typegen` on this client it seems to generate an invalid <MY_PATH> file.

It exports the resources fine with a dollar sign preceding the numbers

```
    namespace Responses {
      export type $200 = Components.Schemas.HospitalUtilization;
      export type $422 = Components.Schemas.HTTPValidationError;
    }
```

But then the generated response does not have the preceding dollarsign, which is invalid typescript and doesn't reference the generated code properly
```
    get(
      parameters?: Parameters<Paths.<MY_PATH>.PathParameters>,
      data?: any,
      config?: AxiosRequestConfig  
    ): OperationResponse<Paths.<MY_PATH>.Responses.200 | Paths.<MY_PATH>.Responses.422>
  }
```

Does anyone else run into this issue? If I find and replace all of these with the preceding dollar sign it seems to work fine.

The other method of generating a client- directly from my openapi.json works as well but I was hoping to switch to the `typegen` method.

**Comment by anttiviljami** [comment]
Seems like this bug was introduced from updating dtsgenerator to the latest version.

**Comment by anttiviljami** [comment]
This is fixed in `openapi-client-axios-typegen@1.1.1`
